@xen-orchestra/backups/_deltaVm.js

```diff
@@ -245,6 +250,10 @@ exports.importDeltaVm = defer(async function importDeltaVm(
   const newVdis = {}
   await asyncMap(Object.keys(vdiRecords), async vdiRef => {
     const vdi = vdiRecords[vdiRef]
+    if (mapVdisSrRefs[vdi.uuid] === null) {
+      return
+    }
+
```

Comment:
I feel like this should be implemented at the producer (`RemoteAdapter.readDeltaVmBackup`) instead of the consumer (`importDeltaVm`) level.